### DISCUSSION TITLE
Anyone knows if this works with the FTXS35BVMB?

### DISCUSSION THREAD

**Discussion by mikekuzak** [revk/xESP32-Faikout#413] [Q&A] [answered]
I have a  older unit from 2003 amd was wondering it I could retrofit it with this. Also would that work with any esp32 board and esphome?

**Comment by alexcf** [comment]
Best thing to do is pop open the cover and see what ports you have...

**Comment by Juanitotje** [comment, marked as answer]
I just installed it in 2 FVXS35BMVB and it works with the Faikin board.
Directly on s21

**Reply by mikekuzak** [reply]
I got connected it to the board. Got power and was able to setup wifi and mqtt. But getting a system down message and these logs.
The board is blinking kinda purple .. 
```
2024-09-23T21:38:24: Received PUBLISH from Faikin:faikin-living-room_DB9418 (d0, q0, r0, m0, 'error/faikin-living-room/comms', ... (33 bytes))
2024-09-23T21:39:59: Received PUBLISH from Faikin:faikin-living-room_DB9418 (d0, q0, r0, m0, 'error/faikin-living-room/comms', ... (38 bytes))
```

**Reply by revk** [reply]
Do you have the payload of those messages, that says what the error is!

Also check connections carefully.
